Source/BlueprintSearchVSExtension/Source/UI/ToolWindows/BlueprintSearchWindow/BlueprintSearchVSWindowVM.cs

```diff
@@ -0,0 +1,152 @@
+using BlueprintSearch.Commands.CommandHandlers;
```

Comment:
// Copyright (C) Coconut Lizard Limited. All rights reserved.
// ---------------------------------------------------------

To top of file